plugins/badges-backend/src/plugin.ts

```diff
@@ -63,6 +63,10 @@ export const badgesPlugin = createBackendPlugin({
             auth,
           }),
         );
+        httpRouter.addAuthPolicy({
+          path: '/',
```

Comment:
I don't think we want this as a blanket for all the plugin right? It's only certain requests that should be exempt?

Reply:
Should it be for '/http' paths?

Reply:
I see, so looking at the code the images come from these routes:
```
'/entity/:entityUuid/:badgeId'
'/entity/:namespace/:kind/:name/badge/:badgeId'
```
according to the router.ts: https://github.com/backstage/backstage/blob/master/plugins/badges-backend/src/service/router.ts

Would you be okay with the path being /entity?

Reply:
I think we should probably just copy the `routeIds` as is, and call twice.

Reply:
Just updated, thanks for helping me as this was my first contribution!